code/modules/integrated_electronics/subtypes/smart.dm

```diff
@@ -74,8 +75,8 @@
 /obj/item/integrated_circuit/smart/advanced_pathfinder
 	name = "advanced pathfinder"
 	desc = "This circuit uses a complex processor for long-range pathfinding."
-	extended_desc = "This circuit uses absolute coordinates for target. A path will be generated taking obstacle input into account, \
-	pathing around any instances of said input. The passkey provided from a card reader is used to create a valid path through doorways."
+	extended_desc = "This circuit uses absolute coordinates for its target. A path will be generated to the target, taking the obstacle input into account, \
```

Comment:
uses absolute coordinates *to find* its target

taking *obstacles* into account